src/gaptime.c

```diff
@@ -131,6 +110,37 @@ Int8 SyNanosecondsSinceEpoch(void)
 }
 
 
+/****************************************************************************
+**
+*F  SyTime()  . . . . . . . . . . . . . . . return time spent in milliseconds
```

Comment:
Any reason to move this code? Note that `SyNanosecondsSinceEpoch` is declared in a header so it ought to be accessible without reordering the code (unless I miss something?)

Reply:
I just didn't think to check if it was in the header and moved it on instinct. I'll move it back.